zkchannels-contract/zkchannel_smartpy_script.py

```diff
@@ -40,89 +39,113 @@ def is_g1_not_zero(val):
 
     def __init__(self, cid, custAddr, merchAddr, custPk, merchPk, custFunding, merchFunding, selfDelay, g2, merchPk0, merchPk1, merchPk2, merchPk3, merchPk4, merchPk5, close_flag):
         self.init(
-                  cid               = cid,
-                  custAddr          = custAddr,
-                  merchAddr         = merchAddr,
-                  custPk            = custPk,
-                  merchPk           = merchPk,
-                  custBal           = sp.mutez(0),
-                  merchBal          = sp.mutez(0),
-                  custFunding       = custFunding,
-                  merchFunding      = merchFunding,
-                  status            = sp.nat(AWAITING_FUNDING),
-                  revLock           = sp.bytes("0x00"),
-                  selfDelay         = selfDelay,
-                  delayExpiry       = sp.timestamp(0),
-                  g2                = g2,
-                  merchPk0          = merchPk0,
-                  merchPk1          = merchPk1,
-                  merchPk2          = merchPk2,
-                  merchPk3          = merchPk3,
-                  merchPk4          = merchPk4,
-                  merchPk5          = merchPk5,
+                  cid               = cid,              # the unique identifier for the channel.
+                  custAddr          = custAddr,         # customer tz1 address
+                  merchAddr         = merchAddr,        # merchant tz1 address
+                  custPk            = custPk,           # customer tezos public key
+                  merchPk           = merchPk,          # merchant tezos public key
+                  custBal           = sp.mutez(0),      # customer's closing balance
+                  merchBal          = sp.mutez(0),      # merchant's closing balance
+                  custFunding       = custFunding,      # customer's initial balance
+                  merchFunding      = merchFunding,     # merchant's initial balance
+                  status            = sp.nat(AWAITING_CUST_FUNDING), # contract status
+                  revLock           = sp.bytes("0x00"), # revLock initialized to 0x00
+                  selfDelay         = selfDelay,        # delay in seconds
+                  delayExpiry       = sp.timestamp(0),  # if the delay is triggered, delayExpiry records when the delay is due to expire
+                  g2                = g2,               # PS pubkey
+                  merchPk0          = merchPk0,         # PS pubkey
+                  merchPk1          = merchPk1,         # PS pubkey
+                  merchPk2          = merchPk2,         # PS pubkey
+                  merchPk3          = merchPk3,         # PS pubkey
+                  merchPk4          = merchPk4,         # PS pubkey
+                  merchPk5          = merchPk5,         # PS pubkey
```

Comment:
I suggest we bring this notation in line as much as possible with what is used in the libzkchannels-crypto repo. That would mean, I'm guessing, but @Ayo could probably weigh in here:
 - `x2` for `merchPk0`,
 - If vectors are a thing, a vector `y2` of length five. Otherwise I guess just ... `y1` through `y5`.

Reply:
Based on how we've named it in the contract and tests, renamed to: `merchPk0` -> `y2s_0` etc, and `merchPk5` -> `x2`